cloud/main.js

```diff
@@ -17,6 +17,17 @@ Parse.Cloud.define("hello", function(request, response) {
     response.success(getAliasName());
 });
 
+Parse.Cloud.define("replayMessages", function(request, response) {
+    var requiredParams = ["chatRoomId", "subkey", "count"];
```

Comment:
im not sure i understand, do you mean the fact that pubnub can limit replayed messages based on start and end times also?  yea, i didnt add anything for this. You think i should? i cant really think of any time we'd use it...